plugins/services/src/js/pages/ServicesPage.js

```diff
@@ -43,6 +45,10 @@ var ServicesPage = React.createClass({
     this.updateCurrentTab();
   },
 
+  componentDidMount() {
+    CosmosPackagesStore.fetchAvailablePackages();
```

Comment:
i think instead of fetching the packages once it would be better to subscribe to changes 🤔 (no idea if `CosmosPackageStore` is built that way tho… 🤷‍♂️ )

Reply:
Packages will not change that often. This solution is not ideal but I guess it is okay.

Reply:
It was a middle ground that was decided on versus fetching it once or subscribing to changes. Open to suggestions.

Reply:
Because packages don't change over time I rather see us fetching only when we need it (which is not often) and caching it.